streamz/dataframe/aggregations.py

```diff
@@ -396,7 +397,7 @@ def grouped(self, df, grouper=None):
 
         g = df.groupby(grouper)
 
-        if self.columns is not None:
+        if self.columns is not None and isinstance(df, pd.DataFrame):
```

Comment:
Why this change?  Was there some limitation in cudf where this didn't work?

Reply:
Yes, pandas GorupBy class has separate classes for DataFrame and Series. And GroupBy DataFrame is subscriptable. But cudf has only one class and is not subscriptable.

Reply:
@skmatti rather than adding code to differentiate between Pandas DataFrames and cuDF DataFrames it would be better to just change the cuDF APIs to match Pandas. In situations like these can you just raise an issue on github? Thanks!

Reply:
Yes, makes sense. This PR([/rapids/ai/#895](https://github.com/rapidsai/cudf/pull/895)) may resolve this.